Lib/unittest/loader.py

```diff
@@ -122,7 +122,7 @@ def loadTestsFromName(self, name, module=None):
 
         The method optionally resolves the names relative to a given module.
         """
-        parts = name.split('.')
+        parts = name.lstrip('.').split('.')
```

Comment:
It's a bit weird to fix this here because `name` is already supposed to be a module name. Can you fix it in `unittest.main._convert_name` instead, where it turns file paths into module names?

Reply:
That's a good point. I fixed this in `Lib/unittest/main.py:_convert_name()`. Should work in windows as well if I am correct.